playwright/package-lock.json

```diff
@@ -9,7 +9,7 @@
       "version": "4.4.1",
       "license": "MPL",
       "dependencies": {
-        "axe-core": "^4.4.2"
+        "axe-core": "^4.4.3"
```

Comment:
Good catch - I must of missed this in a previous PR, when installing `axe-test-fixtures` as we denoted `^` for axe-core it updated it to 4.4.3 within [node_modules](https://github.com/dequelabs/axe-core-maven-html/pull/240/files#diff-d3b2d086922e59583106c513a7abfb49f34cdaf70e13ec300f4b564f84a26fcdR43-R48)